datadog_checks_dev/datadog_checks/dev/tooling/commands/release/testable.py

```diff
@@ -61,21 +61,20 @@ def create_trello_card(client, teams, pr_title, pr_url, pr_body, dry_run):
                 )
                 time.sleep(wait_time)
             else:
-                echo_success('Created card for team {}: '.format(team), nl=False)
-                echo_info(response.json().get('url'))
+                echo_success('Created issue {} for team {}'.format(response.json().get('key'), team))
```

Comment:
Mind if we keep the `blue(...): white(url)`?

Reply:
The Jira API doesn't return the exact link to the issue like Trello does, so I left it out, but I can hardcode the link in like this: 
```
echo_success('Created issue for team {}: '.format(team), nl=False)
echo_info('https://datadoghq.atlassian.net/browse/{}'.format(response.json().get('key')))
```

Reply:
Ah, np then